blazor/treeview/authorization-and-authentication.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: Authorization,Authentication in Blazor TreeView Component | Syncfusion
+description: Checkout and learn here all about Authorization and Authentication in Syncfusion Blazor TreeView component and much more details.
+platform: Blazor
+control: TreeView
+documentation: ug
+---
+
+# Authorization and Authentication in Blazor TreeView Component

```

Comment:
the UG seems incomplete without examples and detailed explanation

Reply:
Done.